chapter_1/Readme.md

```diff
@@ -0,0 +1 @@
+I decided to fix all the errors that rubocop was giving me so I can learn what is considered to be "convention" in writing neat, readable, elegant code.
```

Comment:
Ok, but _how_ did you fix the errors?  i.e `I converted the double quoted lines to single quotes because...`